website/docs/concurrency_control.md

```diff
@@ -2,105 +2,126 @@
 
-In case the lock cannot release during job crash, you can set `hoodie.write.lock.filesystem.expire` (lock will never expire by default). You may also delete lock file manually in such situation.
+In case of multi-writing in Hudi, the locks are acquired on the Hudi table for a very short duration during specific phases (such as just before committing the writes or before scheduling table services) instead of locking for the entire span of time. This approach allows multiple writers to work on the same table simultaneously, increasing concurrency and avoids conflicts.
 
-**`Zookeeper`** based lock provider
+There are 4 different lock providers that require different configurations to be set. Please refer to comprehensive locking configs [here](https://hudi.apache.org/docs/next/configurations#LOCK).
 
+#### Zookeeper based lock provider
 ```
 hoodie.write.lock.provider=org.apache.hudi.client.transaction.lock.ZookeeperBasedLockProvider
-hoodie.write.lock.zookeeper.url
-hoodie.write.lock.zookeeper.port
-hoodie.write.lock.zookeeper.lock_key
-hoodie.write.lock.zookeeper.base_path
 ```
+Following are the basic configs required to setup this lock provider:
 
-**`HiveMetastore`** based lock provider
+| Config Name| Default| Description                                                                                                                                                                                                |
+| ---------------------------------------------------------------------------- | ------------------------ |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| hoodie.write.lock.zookeeper.base_path   | N/A **(Required)**   | The base path on Zookeeper under which to create lock related ZNodes. This should be same for all concurrent writers to the same table<br /><br />`Config Param: ZK_BASE_PATH`<br />`Since Version: 0.8.0` |
+| hoodie.write.lock.zookeeper.port        | N/A **(Required)**   | Zookeeper port to connect to.<br /><br />`Config Param: ZK_PORT`<br />`Since Version: 0.8.0`                                                                                                               |
+| hoodie.write.lock.zookeeper.url         | N/A **(Required)**   | Zookeeper URL to connect to.<br /><br />`Config Param: ZK_CONNECT_URL`<br />`Since Version: 0.8.0`                                                                                                         |
```

Comment:
hoodie.write.lock.zookeeper.lock_key
hoodie.write.lock.zookeeper.base_path are missing

Reply:
hoodie.write.lock.zookeeper.lock_key would assume table name if not set explicitly. So kept this list of configs to minimal number of ones. 
The `hoodie.write.lock.zookeeper.base_path` is here - https://github.com/apache/hudi/pull/9372/files#diff-9b6f64b1c2c1b2e6b9165165ee949233d9df1f6bd3ce0771b6523f76c16d988cR97